tests/test_cuboid.py

```diff
@@ -0,0 +1,61 @@
+#!/usr/bin/env python
+
+import pytest
+import unittest
+from ddt import ddt, data, unpack
+
+import numpy as np
+from broni.shapes.primitives import Cuboid
+
+
+@ddt
+class TestCuboid(unittest.TestCase):
+    def setUp(self):
+        pass
+
+    def tearDown(self):
+        pass
+
+    def test_invalid(self):
+        with pytest.raises(ValueError) as e:
+            assert Cuboid(0, 0, 0, 0, 0, 0)
```

Comment:
does it work with Cuboid(10,10,10,3,3,3) ?

Reply:
Yes, test added.